src/main/kotlin/omok/domain/Hand.kt

```diff
@@ -0,0 +1,11 @@
+package omok.domain
+
+class Hand {
+    private var _stones = mutableListOf<Stone>()
+    val stones
+        get() = _stones.toList()
```

Comment:
```suggestion
    private val _stones = mutableListOf<Stone>()
    val stones: List<Stone>
        get() = _stones.toList()
```
List, Set과 같은 컬렉션은, 변수의 주솟값을 변경하지 않고도 데이터를 추가/제거 할 수 있어요.
해서, 컬렉션은 var보다는 val 사용을 권장합니다.

Reply:
넵! val로 수정했습니다!